examples/partial_labels.py

```diff
@@ -0,0 +1,182 @@
+"""
+TODO How to revert to previous model (in the same model group) if necessary?
+
+Outside of Script
+* Expose API or script for comparing model metrics between original model and PL model
+    * Determine if we should add new value matching metrics into somewhere more "official"
```

Comment:
100% should be doing this...
In meantime, metrics module may do (pretty much) everything you need already